pennylane_qiskit/qiskit_device.py

```diff
@@ -154,25 +162,26 @@ def __init__(self, wires, provider, backend, shots=1024, **kwargs):
 
             self.shots = 1024
 
-        self._backend = None
-
         self.provider = provider
```

Comment:
I don't want to remove `provider` from `self` because users might depend on it, but it's not used outside of the `__init__` method anymore, so I kinda do...